src/components/Book.js

```diff
@@ -13,28 +13,38 @@ const Book = (props) => {
   };
 
   return (
-    <li>
-      <div>
-        <p>
-          <span>Book title :</span>
-          {title}
-        </p>
+    <li className="book">
+      <div className="left-block">
+        <div className="book-info">
+          <span className="category">{category}</span>
+          <span className="title">{title}</span>
+          <span className="author">{author}</span>
+        </div>
+        <div className="list-btn">
+          <button type="button">Comments </button>
+          |
+          <button
+            type="button"
+            onClick={handleRemoveClick}
+          >
+            Remove
+          </button>
+          |
+          <button type="button">Edit</button>
+        </div>
       </div>
-      <div>
-        <p>
-          <span>Author :</span>
-          {author}
-        </p>
+      <div className="reading-progress">
+        <div className="progress" />
+        <div className="progress-percentage">
+          <span className="percentage">76%</span>
+          <span className="completed">Completed</span>
+        </div>
       </div>
```

Comment:
- [ ] As per Zeplin design, there is a vertical rule between the percentage completion and the Chapter number as seen in the screenshot below. This vertical rule is missing in your design
![Screen Shot 2023-09-02 at 6 12 19 AM](https://github.com/tajemouti/bookstore/assets/62849872/a67b6d61-565a-47b3-8f23-b3eb357a9912)